components/server-side-render/index.js

```diff
@@ -42,9 +42,9 @@ export class ServerSideRender extends Component {
 		this.isStillMounted = false;
 	}
 
-	componentWillReceiveProps( nextProps ) {
-		if ( ! isEqual( nextProps, this.props ) ) {
-			this.fetch( nextProps );
+	componentDidUpdate( prevProps ) {
+		if ( ! isEqual( prevProps, this.props ) ) {
```

Comment:
Unrelated, should we use `isShallowEqual` like in other places when comparing props?

Reply:
Probably, I'm not familiar with this component to tell though. Also, it looks like it's not being used in Gutenberg at all.

Reply:
>Also, it looks like it's not being used in Gutenberg at all.

Can you elaborate on this point?

Reply:
The `ServerSideRender` component is not used in Gutenberg.